conda_build/api.py

```diff
@@ -15,13 +15,16 @@
 #    to conda-build's functionality.
 import os
 import sys
+from collections.abc import Iterable
```

Comment:
For anyone else who might wonder why this isn't under `typing`: https://stackoverflow.com/questions/52827463/collections-iterable-vs-typing-iterable-in-type-annotation-and-checking-for-iter

Reply:
📚 `:the_more_you_know:`

Thanks for linking to that!